plugins/nvm/README.md

```diff
@@ -21,14 +21,32 @@ These settings should go in your zshrc file, before Oh My Zsh is sourced:
   [Homebrew is installed in `/opt/homebrew`](https://docs.brew.sh/Installation). To get the directory where
   nvm has been installed, regardless of chip architecture, use `NVM_HOMEBREW=$(brew --prefix nvm)`.
 
-- **`NVM_LAZY`**: if you want the plugin to defer the load of nvm to speed-up the start of your zsh session,
-  set `NVM_LAZY` to `1`. This will source nvm script only when using it, and will create a function for `node`,
-  `npm`, `pnpm`, `yarn`, and the command(s) specified by `NVM_LAZY_CMD`, so when you call either of them,
-  nvm will be loaded and run with default version.
+## Customization
 
-- **`NVM_LAZY_CMD`**: if you want additional command(s) to trigger lazy loading of nvm, set `NVM_LAZY_CMD` to
-  the command or an array of the commands.
+#### Lazy startup
 
-- **`NVM_AUTOLOAD`**: if `NVM_AUTOLOAD` is set to `1`, the plugin will automatically load a node version when
-  if finds a [`.nvmrc` file](https://github.com/nvm-sh/nvm#nvmrc) in the current working directory indicating
-  which node version to load.
+This option will help you to defer nvm's load until you use it to speed-up your zsh startup. This will source
+nvm script only when using it, and will create a function for `node`, `npm`, `pnpm`, `yarn`, and the
+command(s) specified by `lazy-cmd` option, so when you call either of them, nvm will be loaded and run with
+default version. To enable it, you can add this snippet to your zshrc, before Oh My Zsh is sourced:
+
+```zsh
+zstyle ':omz:plugins:nvm' lazy true
```

Comment:
We should standardize on what value we suggest for boolean options. I remember the `ssh-agent` plugin uses `lazy yes` instead of `true`, and we should look at how we're doing it on other plugins.

Reply:
Other plugins are using `yes` as the boolean in the README, but `yarn` plugin is recommending `false`. We can move everything to `yes` and `no` (even though both are going to work the same way and are compatible).